install/install.queries.php

```diff
@@ -1497,8 +1508,8 @@ function encryptFollowingDefuse($message, $ascii_key)
 define("DB_CONNECT_OPTIONS", array(
     MYSQLI_OPT_CONNECT_TIMEOUT => 10
 ));
-define("SECUREPATH", "' . $securePath . '");
-define("SECUREFILE", "' . $filesecure. '");
+define("SECUREPATH", "' . $session_sk_path . '");
```

Comment:
Wouldn't a user who mistakenly inserted a quote into the form find themselves stuck with a HTTP 500 error?

Reply:
You are right.
I will just add an str_replace but a full control should be done another way.